libs/sdk-backend-tiger/src/backend/workspace/dashboards/index.ts

```diff
@@ -420,15 +421,84 @@ export class TigerWorkspaceDashboards implements IWorkspaceDashboardsService {
                 visualExportRequest,
             });
 
-            return await this.handleExportResultPolling(client, {
+            return await this.handleExportResultPolling(client, "application/pdf", {
                 workspaceId: this.workspace,
                 exportId: pdfExport?.data?.exportResult,
             });
         });
     };
 
+    public exportDashboardToPresentation = async (
+        dashboardRef: ObjRef,
+        format: "pdf" | "pptx",
+        filters?: FilterContextItem[],
+    ): Promise<IExportResult> => {
+        const dashboardId = await objRefToIdentifier(dashboardRef, this.authCall);
+
+        // skip all time date filter from stored filters, when missing, it's correctly
+        // restored to All time during the load later
+        const withoutAllTime = (filters || []).filter((f) => !isAllTimeDashboardDateFilter(f));
+
+        return this.authCall(async (client) => {
+            const dashboardResponse = await client.entities.getEntityAnalyticalDashboards(
+                {
+                    workspaceId: this.workspace,
+                    objectId: dashboardId,
+                },
+                {
+                    headers: jsonApiHeaders,
+                },
+            );
+
+            const { title } = convertDashboard(dashboardResponse.data);
+            const slideshowExportRequest = {
+                fileName: title,
+                dashboardId,
+                format: format.toUpperCase() as SlideshowExportRequestFormatEnum,
```

Comment:
I would vote for making the convertor explicit so there are not any unnecessary surprises in the future